### ISSUE TITLE
logstash does not start via ssh remote command

### ISSUE THREAD

**Issue by BadBossy** [elastic/logstash#3076]
Hello,

I've got a problem with logstash. I can't start it via ssh remote command. It doesn't throw an error message. My for-loop goes over every server and does "/etc/init.d/logstash start" but it doesn't start logstash, nothing happens.

Anyone got a fix for this?

**Comment by lmorfitt** [comment]
Hello,

Please provide details of your OS, logstash version and the example of the command you are running.

thanks

**Comment by BadBossy** [comment]
My ServerCluster uses:
--OS: Centos6.6 (up2date)

--Logstash: yum list installed | grep logstash
                  logstash.noarch                   1.4.2-1_2c0f5a1        @logstash-1.4

--Command: r=$(ssh $serv_ssh_option $serv_user@$server $serv_ssh_source $cmd 2>&1)
                     $cmd=/etc/init.d/logstash start
                     $serv_ssh_option=-t -q  -o ConnectTimeout=1 -o PasswordAuthentication=no -o RhostsAuthentication=no -o StrictHostKeyChecking=no


--In short: ssh user@maschine /etc/init.d/logstash start

And logstash doesn't start on any maschine.

**Comment by lmorfitt** [comment]
Can you login to the box its self and run the init script? Does it start?

**Comment by BadBossy** [comment]
Yes, if I start it local it works.. just remote doesn't work.

**Comment by BadBossy** [comment]
Someone got a new Information for me?

**Comment by ianmacl** [comment]
I have seen this issue and worked around it by not creating a pty (psuedoterminal).  This defied logic and I had wanted to go back and delve deeper into the issue, but that resolved the issue for me.

**Comment by asmolov** [comment]
I'm having the same issue.
When I run `service logstash restart` locally, it all works well. But if I do it through ssh, it will notify 'logstash started.', but it won't.
If I check logstash status, I will get 'logstash is not running'.
I use init script from here:
https://github.com/elastic/logstash/blob/master/pkg/logstash.sysv

**Comment by ph** [comment]
I don't think this is an issue anymore https://github.com/elastic/logstash/issues/3005

**Comment by guyboertje** [comment]
We have rewritten this build run scripts. I am not sure this error is still around. Reopen if it is.
